01-import_and_filter.py

```diff
@@ -31,11 +32,11 @@ def run_filter(subject):
     raws = []
     for run in config.runs:
 
-        # read bad channels for run from config
-        if run:
-            bads = config.bads[subject][run]
-        else:
-            bads = config.bads[subject]
+         # read bad channels for run from config
```

Comment:
@DraganaMana you should not have to comment that out. Does it break if you put no runs/ bad channels?

Reply:
Yes, we tried a few options: omitting bads, leaving it bads = dict() , bads = {}, but we still get an error such as: 
  File "C:/Users/Dragana/Repositories/CogBrainDyn_MEG_Pipeline/01-import_and_filter.py", line 39, in run_filter
    bads = config.bads[subject]

Reply:
I tried now by defining the MISC 001, MISC 002 as bad channels, and it's still not working.

Reply:
It needs t be a dict with the subject name and the bad channels:
bads = dict(my_subject=['MEG 2443', 'EEG 053'])
Please try that and put it back, you will need it. 
For the new data, you can put bad channels per run.

Reply:
@Izem0